CoronaCare/package.json

```diff
@@ -21,6 +21,8 @@
     "react-native-elements": "^1.2.7",
     "react-native-linear-gradient": "^2.5.6",
     "react-native-navigation": "^6.4.0",
+    "react-native-swipe-list-view": "^3.1.0",
+    "react-native-swipeable-flat-list": "^0.1.2",
```

Comment:
should be removed

Reply:
referring to react-native-swipeable-flat-list, but its okay if its there, wont be an issue not removing it